helm/coredns-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 1.6.5
+appVersion: 1.6.9
```

Comment:
actual tag reference https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L31 has to be changed too

Reply:
Thanks @sslavic . What I am trying to accomplish is to test a custom coredns image (hackathon project) with our GS helm chart values in a TC. The changes I have made:

- Create new test branch "AddCustomPlugin" and using this PR to track changes with @stone-z .
    - change `chart.yaml` to the same upstream version used in the "new" image
    - change  `values.yml` values `giantswarm/coredns:1.6.5` to `benrain/blacklist:v1` newly compiled coredns image on dockerhub
    - update `.cicleci/config.yml` to put this .tar for custom helm chart into `default-test-catalog` in order for TC to be able to pull the new test image

Reply:
ah, clear, thanks for heads up!
maybe consider also contributing actual coredns 1.6.9 update too :)